packages/sdk/src/utils/chain.ts

```diff
@@ -0,0 +1,113 @@
+import { BigNumber, ethers } from 'ethers'
+import { EthereumProvider } from 'hardhat/types'
+
+import { Chain, L1Chain } from '../interfaces'
+
+export const OptimismKovan = {
+  chainId: Chain.OPTIMISM_KOVAN,
+  chainName: 'OptimismKovan',
+  isTestChain: true,
+  isLocalChain: false,
+  multicallAddress: '0xE71bf4622578c7d1526A88CD3060f03030E99a04',
```

Comment:
Do we use this multicall address? Because multicall v3 is out and its at a universal address

Reply:
If we are using multicall, we should use this: https://twitter.com/msolomon44/status/1501631244844498944 and just include the ABI in our source code with an ethers contract object that can be easily instantiated

Reply:
@tynes We might not.  I copy pasted this from synthetix here https://github.com/TrueFiEng/useDApp/blob/b8e5c8196ac5ae62544dfbf5c8afd04338325c69/packages/core/src/model/chain/optimism.ts

I see that is the same address actually deployed by hardhat here https://github.com/withtally/dao.new/blob/72cb3b1a30ee889557fc67dbee3f06b0d782f5e0/packages/hardhat/deployed_contracts/deployed_contracts.arbitrum-rinkeby.json

So there is a chance that address is just copy-paste-pasta

Reply:
We should prune this stuff as to not add dead code to the repo :)

Reply:
Using that universal multicall seems like a good idea btw (not in this pr but seperate pr)

Reply:
Oh this is useDapp not synthetix.  We will want to do a pr to useDapp too.

Reply:
I'll make a note to fix useDapp whenever we switch (or decide to not switch) to goerli